src/content/self-hosted/manage/troubleshooting.mdx

```diff
@@ -5,6 +5,45 @@ sidebar_label: Troubleshooting
 id: troubleshooting
 ---
 
+## How to extract logs from Okteto when asking for help
+
+When asking for assistance from Okteto Support, please use the [`okteto logs` command](reference/okteto-cli.mdx#logs) to fetch the logs from your Development Environment.
+
+## Understanding the state of your app: why endpoints aren't present in the CLI or UI
```

Comment:
This page is for troubleshooting SH. But this is useful for developers. 
I think better options for this content are:
- Okteto Endoints: https://www.okteto.com/docs/core/endpoints/automatic-ssl/
- FAQ: https://www.okteto.com/docs/reference/faqs/

Reply:
@codyjlandstrom this comment still applies. This page is for SH troubleshooting, but this content is generic, it's useful for SaaS and also for developers, not only for the platform team

Reply:
I've added an introduction and linked to our FAQ guide there so it's clear